.devcontainer/devcontainer.json

```diff
@@ -21,28 +21,29 @@
            },
            // Add the IDs of extensions you want installed when the container is created.
            "extensions": [
-           "vadimcn.vscode-lldb",
-           "mutantdino.resourcemonitor",
-           "rust-lang.rust-analyzer",
-           "tamasfe.even-better-toml",
-           "serayuzgur.crates",
-           "timonwong.shellcheck",
-           "eamodio.gitlens",
-           "streetsidesoftware.code-spell-checker",
-           "jebbs.plantuml",
-           "ms-python.python",
-           "ms-python.vscode-pylance",
-           "yzhang.markdown-all-in-one",
-           "zxh404.vscode-proto3",
-           "bierner.markdown-preview-github-styles",
-           "hediet.vscode-drawio",
-           "redhat.vscode-yaml",
-           "bierner.markdown-mermaid",
-           "d-biehl.robotcode",
-           "bianxianyang.htmlplay",
-           "DavidAnson.vscode-markdownlint",
-           "EditorConfig.EditorConfig"
-           ]
+                "vadimcn.vscode-lldb",
+                "mutantdino.resourcemonitor",
+                "rust-lang.rust-analyzer",
+                "tamasfe.even-better-toml",
+                "serayuzgur.crates",
+                "timonwong.shellcheck",
+                "eamodio.gitlens",
+                "streetsidesoftware.code-spell-checker",
+                "jebbs.plantuml",
+                "ms-python.python",
+                "ms-python.vscode-pylance",
+                "yzhang.markdown-all-in-one",
+                "zxh404.vscode-proto3",
+                "bierner.markdown-preview-github-styles",
+                "hediet.vscode-drawio",
+                "redhat.vscode-yaml",
+                "bierner.markdown-mermaid",
+                "d-biehl.robotcode",
+                "bianxianyang.htmlplay",
+                "DavidAnson.vscode-markdownlint",
+                "EditorConfig.EditorConfig",
+                "PKief.material-icon-theme"
```

Comment:
"PKief.material-icon-theme" is the only new one.

Reply:
I have removed it.